### PR TITLE
Add scripts for networkpolicy testing

### PR DESCRIPTION
**Pull request by smalleni**
Signed-off-by: Sai Sindhur Malleni <smalleni@redhat.com>

### CODE REVIEW

**Comment by smalleni** [comment]
@rsevilla87 @jtaleric PTAL

**Comment by dry923** [comment]
@smalleni is there a sample run you can point to or some result from the added test. [we really need a CI for this repo]

File: workloads/network-perf/common.sh [hunk 1]
```diff
@@ -38,12 +38,21 @@ export_defaults() {
   latency_tolerance=${LATENCY_TOLERANCE:=5}
   export client_server_pairs=(1 2 4)
   export pin=false
+  networkpolicy=${NETWORK_POLICY:=false}
```
Comment (mohit-sheth) [inline workloads/network-perf/common.sh:41]: just add export here?

File: workloads/network-perf/common.sh [hunk 2]
```diff
@@ -38,12 +38,21 @@ export_defaults() {
...
     export client=$(oc get nodes -l node-role.kubernetes.io/worker -o jsonpath='{range .items[*]}{ .metadata.labels.kubernetes\.io/hostname}{"\n"}{end}' | tail -n 1)
     export pin=true
   fi
+	
+  if [ ${networkpolicy} = true ]
+  then
+     export networkpolicy=true
+  else
+     export networkpolicy=false
+  fi
```
Comment (mohit-sheth) [inline workloads/network-perf/common.sh:55]: redundant if https://github.com/cloud-bulldozer/e2e-benchmarking/pull/80/files#r543453524 works

**Review by mohit-sheth — APPROVED** [review]
LGTM

**Review by rsevilla87 — APPROVED** [review]
LGTM!

**Pull request merged**